java/java-hello-world/skaffold.yaml

```diff
@@ -5,8 +5,13 @@ build:
     sha256: {}
   # defines where to find the code at build time and where to push the resulting image
   artifacts:
-  - image: java-hello-world
-    context: .
+    - image: java-hello-world
+      context: .
+      # Comment out the jibMaven line below to use the Dockerfile based build.
+      # To learn more about how jib build Java containers visit
+      # https://github.com/GoogleContainerTools/jib
+      jibMaven: {}
```

Comment:
Will Cloud Code download Jib on the machines where it's not installed? Will it work both on Windows, macOS and Linux the same?

Reply:
yes, jib is just a maven/gradle plugin, those systems handle downloading everything required.